scss/components/organisms/_footer.scss

```diff
@@ -0,0 +1,27 @@
+@charset "UTF-8";
+
+//
+// Footer
+//
+// The global footer.
+//
+// Markup: ../../templates/components/organisms/footer/footer.twig
+//
+// Style guide: Organisms.Footer
+//
+
+.footer-wrapper {
```

Comment:
This class is too generically named and should be renamed semantically to something like `global-footer`